### PR TITLE
clean: Update CircleCI to use codacy-plugins-test orb

### PR DESCRIPTION
**Pull request by lolgab**
- Bump Scala to 2.13.1
- Bump sbt to 1.3.5
- Use separate base docker image
- Use codacy-sbt-plugin
- Check scalafmt in CI

### CODE REVIEW

File: src/main/scala/codacy/rubocop/RubocopResult.scala [hunk 1]
```diff
@@ -4,9 +4,21 @@ import play.api.libs.json.{Format, JsString, JsValue, Json}
...
 case class RubocopLocation(line: Int, column: JsValue, length: JsValue)
 
-case class RubocopMetadata(rubocop_version: JsValue, ruby_engine: JsValue, ruby_version: JsValue, ruby_patchlevel: JsValue, ruby_platform: JsValue)
-
-case class RubocopOffense(severity: JsValue, message: JsString, cop_name: JsString, corrected: JsValue, location: RubocopLocation)
+case class RubocopMetadata(
+    rubocop_version: JsValue,
+    ruby_engine: JsValue,
+    ruby_version: JsValue,
+    ruby_patchlevel: JsValue,
```
Comment (rtfpessoa) [inline src/main/scala/codacy/rubocop/RubocopResult.scala]: Codacy Issue found: [Field name 'ruby_patchlevel' does not match the regular expression '^[a-z][A-Za-z0-9]*$'](https://app.codacy.com/gh/codacy/codacy-rubocop/pullRequest?prid=4769612)

File: src/main/scala/codacy/rubocop/RubocopResult.scala [hunk 2]
```diff
@@ -4,9 +4,21 @@ import play.api.libs.json.{Format, JsString, JsValue, Json}
...
+    ruby_platform: JsValue
+)
+
+case class RubocopOffense(
+    severity: JsValue,
+    message: JsString,
+    cop_name: JsString,
```
Comment (rtfpessoa) [inline src/main/scala/codacy/rubocop/RubocopResult.scala]: Codacy Issue found: [Field name 'cop_name' does not match the regular expression '^[a-z][A-Za-z0-9]*$'](https://app.codacy.com/gh/codacy/codacy-rubocop/pullRequest?prid=4769612)

File: src/main/scala/codacy/rubocop/RubocopResult.scala [hunk 3]
```diff
@@ -4,9 +4,21 @@ import play.api.libs.json.{Format, JsString, JsValue, Json}
 
 case class RubocopLocation(line: Int, column: JsValue, length: JsValue)
 
-case class RubocopMetadata(rubocop_version: JsValue, ruby_engine: JsValue, ruby_version: JsValue, ruby_patchlevel: JsValue, ruby_platform: JsValue)
-
-case class RubocopOffense(severity: JsValue, message: JsString, cop_name: JsString, corrected: JsValue, location: RubocopLocation)
+case class RubocopMetadata(
+    rubocop_version: JsValue,
+    ruby_engine: JsValue,
+    ruby_version: JsValue,
```
Comment (rtfpessoa) [inline src/main/scala/codacy/rubocop/RubocopResult.scala]: Codacy Issue found: [Field name 'ruby_version' does not match the regular expression '^[a-z][A-Za-z0-9]*$'](https://app.codacy.com/gh/codacy/codacy-rubocop/pullRequest?prid=4769612)

File: src/main/scala/codacy/rubocop/RubocopResult.scala [hunk 4]
```diff
@@ -4,9 +4,21 @@ import play.api.libs.json.{Format, JsString, JsValue, Json}
 
 case class RubocopLocation(line: Int, column: JsValue, length: JsValue)
 
-case class RubocopMetadata(rubocop_version: JsValue, ruby_engine: JsValue, ruby_version: JsValue, ruby_patchlevel: JsValue, ruby_platform: JsValue)
-
-case class RubocopOffense(severity: JsValue, message: JsString, cop_name: JsString, corrected: JsValue, location: RubocopLocation)
+case class RubocopMetadata(
+    rubocop_version: JsValue,
```
Comment (rtfpessoa) [inline src/main/scala/codacy/rubocop/RubocopResult.scala]: Codacy Issue found: [Field name 'rubocop_version' does not match the regular expression '^[a-z][A-Za-z0-9]*$'](https://app.codacy.com/gh/codacy/codacy-rubocop/pullRequest?prid=4769612)

File: src/main/scala/codacy/rubocop/RubocopResult.scala [hunk 5]
```diff
@@ -4,9 +4,21 @@ import play.api.libs.json.{Format, JsString, JsValue, Json}
...
+    ruby_engine: JsValue,
+    ruby_version: JsValue,
+    ruby_patchlevel: JsValue,
+    ruby_platform: JsValue
```
Comment (rtfpessoa) [inline src/main/scala/codacy/rubocop/RubocopResult.scala]: Codacy Issue found: [Field name 'ruby_platform' does not match the regular expression '^[a-z][A-Za-z0-9]*$'](https://app.codacy.com/gh/codacy/codacy-rubocop/pullRequest?prid=4769612)

File: src/main/scala/codacy/rubocop/RubocopResult.scala (hunk 1 above)
Comment (rtfpessoa) [inline src/main/scala/codacy/rubocop/RubocopResult.scala]: Codacy Issue found: [Field name 'ruby_patchlevel' does not match the regular expression '^[a-z][A-Za-z0-9]*$'](https://app.codacy.com/gh/codacy/codacy-rubocop/pullRequest?prid=4769612)

File: src/main/scala/codacy/rubocop/RubocopResult.scala (hunk 5 above)
Comment (rtfpessoa) [inline src/main/scala/codacy/rubocop/RubocopResult.scala]: Codacy Issue found: [Field name 'ruby_platform' does not match the regular expression '^[a-z][A-Za-z0-9]*$'](https://app.codacy.com/gh/codacy/codacy-rubocop/pullRequest?prid=4769612)

File: src/main/scala/codacy/rubocop/RubocopResult.scala (hunk 3 above)
Comment (rtfpessoa) [inline src/main/scala/codacy/rubocop/RubocopResult.scala]: Codacy Issue found: [Field name 'ruby_version' does not match the regular expression '^[a-z][A-Za-z0-9]*$'](https://app.codacy.com/gh/codacy/codacy-rubocop/pullRequest?prid=4769612)

File: src/main/scala/codacy/rubocop/RubocopResult.scala (hunk 4 above)
Comment (rtfpessoa) [inline src/main/scala/codacy/rubocop/RubocopResult.scala]: Codacy Issue found: [Field name 'rubocop_version' does not match the regular expression '^[a-z][A-Za-z0-9]*$'](https://app.codacy.com/gh/codacy/codacy-rubocop/pullRequest?prid=4769612)

File: src/main/scala/codacy/rubocop/RubocopResult.scala (hunk 2 above)
Comment (rtfpessoa) [inline src/main/scala/codacy/rubocop/RubocopResult.scala]: Codacy Issue found: [Field name 'cop_name' does not match the regular expression '^[a-z][A-Za-z0-9]*$'](https://app.codacy.com/gh/codacy/codacy-rubocop/pullRequest?prid=4769612)

File: src/main/scala/codacy/rubocop/RubocopResult.scala (hunk 1 above)
Comment (rtfpessoa) [inline src/main/scala/codacy/rubocop/RubocopResult.scala]: Codacy Issue found: [Field name 'ruby_patchlevel' does not match the regular expression '^[a-z][A-Za-z0-9]*$'](https://app.codacy.com/gh/codacy/codacy-rubocop/pullRequest?prid=4769612)

File: src/main/scala/codacy/rubocop/RubocopResult.scala (hunk 5 above)
Comment (rtfpessoa) [inline src/main/scala/codacy/rubocop/RubocopResult.scala]: Codacy Issue found: [Field name 'ruby_platform' does not match the regular expression '^[a-z][A-Za-z0-9]*$'](https://app.codacy.com/gh/codacy/codacy-rubocop/pullRequest?prid=4769612)

File: src/main/scala/codacy/rubocop/RubocopResult.scala (hunk 3 above)
Comment (rtfpessoa) [inline src/main/scala/codacy/rubocop/RubocopResult.scala]: Codacy Issue found: [Field name 'ruby_version' does not match the regular expression '^[a-z][A-Za-z0-9]*$'](https://app.codacy.com/gh/codacy/codacy-rubocop/pullRequest?prid=4769612)

File: src/main/scala/codacy/rubocop/RubocopResult.scala (hunk 4 above)
Comment (rtfpessoa) [inline src/main/scala/codacy/rubocop/RubocopResult.scala]: Codacy Issue found: [Field name 'rubocop_version' does not match the regular expression '^[a-z][A-Za-z0-9]*$'](https://app.codacy.com/gh/codacy/codacy-rubocop/pullRequest?prid=4769612)

File: src/main/scala/codacy/rubocop/RubocopResult.scala (hunk 2 above)
Comment (rtfpessoa) [inline src/main/scala/codacy/rubocop/RubocopResult.scala]: Codacy Issue found: [Field name 'cop_name' does not match the regular expression '^[a-z][A-Za-z0-9]*$'](https://app.codacy.com/gh/codacy/codacy-rubocop/pullRequest?prid=4769612)

File: build.sbt [hunk 6]
```diff
@@ -1,100 +1,68 @@
...
-}
+mappings in Universal ++= (resourceDirectory in Compile).map { resourceDir: File =>
+  val src = resourceDir / "docs"
+  val dest = "/docs"
+
+  val docFiles = for {
+    path <- src.allPaths.get if !path.isDirectory
+  } yield path -> path.toString.replaceFirst(src.toString, dest)
+
+  val rubyFiles = Seq(
```
Comment (josemiguelmelo) [inline build.sbt]: Do you need to add these rubyFiles? Aren't they added to the base image?

File: src/main/scala/codacy/rubocop/RubocopResult.scala (hunk 1 above)
Comment (rtfpessoa) [inline src/main/scala/codacy/rubocop/RubocopResult.scala]: Codacy Issue found: [Field name 'ruby_patchlevel' does not match the regular expression '^[a-z][A-Za-z0-9]*$'](https://app.codacy.com/gh/codacy/codacy-rubocop/pullRequest?prid=4769612)

File: src/main/scala/codacy/rubocop/RubocopResult.scala (hunk 5 above)
Comment (rtfpessoa) [inline src/main/scala/codacy/rubocop/RubocopResult.scala]: Codacy Issue found: [Field name 'ruby_platform' does not match the regular expression '^[a-z][A-Za-z0-9]*$'](https://app.codacy.com/gh/codacy/codacy-rubocop/pullRequest?prid=4769612)

File: src/main/scala/codacy/rubocop/RubocopResult.scala (hunk 3 above)
Comment (rtfpessoa) [inline src/main/scala/codacy/rubocop/RubocopResult.scala]: Codacy Issue found: [Field name 'ruby_version' does not match the regular expression '^[a-z][A-Za-z0-9]*$'](https://app.codacy.com/gh/codacy/codacy-rubocop/pullRequest?prid=4769612)

File: src/main/scala/codacy/rubocop/RubocopResult.scala (hunk 4 above)
Comment (rtfpessoa) [inline src/main/scala/codacy/rubocop/RubocopResult.scala]: Codacy Issue found: [Field name 'rubocop_version' does not match the regular expression '^[a-z][A-Za-z0-9]*$'](https://app.codacy.com/gh/codacy/codacy-rubocop/pullRequest?prid=4769612)

File: src/main/scala/codacy/rubocop/RubocopResult.scala (hunk 2 above)
Comment (rtfpessoa) [inline src/main/scala/codacy/rubocop/RubocopResult.scala]: Codacy Issue found: [Field name 'cop_name' does not match the regular expression '^[a-z][A-Za-z0-9]*$'](https://app.codacy.com/gh/codacy/codacy-rubocop/pullRequest?prid=4769612)

File: build.sbt (hunk 6 above)
Comment (lolgab) [reply build.sbt]: Right! ✅ Removed

**Pull request merged**